lib/common.js

```diff
@@ -15,10 +15,10 @@
 
 // and
 
-// module.exports = {
-//   property1 : value1,
-//   property2 : value2
-// };
+module.exports = {
```

Comment:
We were looking for custom functions to be written and exported for this exercise.